src/select/styles/example.m.css

```diff
@@ -0,0 +1,28 @@
+@import '../../common/styles/variables.css';
+
+.option {
```

Comment:
I think most of the example page CSS can go away. Tom is pulling out theme styles from structural styles at dojo/widgets#77. These aesthetic styles could go into the theme CSS, and any minor page styling could just be done inline. I think we should try to keep `styles` clean of example-specific stuff.